pkg/operation/botanist/component/istio/monitoring.go

```diff
@@ -170,15 +170,15 @@ metric_relabel_configs:
 metrics_path: /stats/prometheus
 kubernetes_sd_configs:
 - role: endpoints
-  namespaces:
-    names: [ ` + v1beta1constants.DefaultSNIIngressNamespace + ` ]
+  selectors:
+   label: istio=ingressgateway
```

Comment:
Shouldn't it be two space indentation?

```suggestion
    label: istio=ingressgateway
```

Reply:
fixed